addons/escoria-core/game/core-scripts/behaviors/esc_movable.gd

```diff
@@ -395,48 +395,34 @@ func _is_angle_in_interval(
 		angle = 360
 	var start_angle = wrapi(direction_angle.angle_start, 0, 360)
 	var angle_area = direction_angle.angle_size
-	var end_angle = wrapi(direction_angle.angle_start + angle_area, 0, 360) 
+	var end_angle = wrapi(direction_angle.angle_start + angle_area, 0, 360)
+	if start_angle >= 0 and end_angle == 0:
```

Comment:
Doesn't this so happen in _angle_is_between?

Reply:
Fixed